.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Pull Request Title Standard
+For pull requests, include the issue number and a brief description of the change:
+Use `#<issue_number> | prefix followed by the commit type and a concise description`.
+
+### Description of Changes
+(Provide a summary of the changes made in this pull request. Explain why these changes are necessary and how they improve the project.)
+
+### Related Issue
+(Link to the related issue if applicable, e.g., #42)
+
+### Type of Change
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Improvement
+- [ ] Documentation Update
+- [ ] Other (Please specify)
+
+### Checklist
+- [ ] Documentation updated
+- [ ] Tests added
+- [ ] Code style adheres to standards
+- [ ] Build passes
+- [ ] Lint checks pass
+- [ ] Relevant issue is linked
+
+### Additional Notes
+(Provide any additional context or comments related to this pull request. For example, discuss any potential impacts on other parts of the codebase, or mention any known issues.)
+
+### Screenshots (if applicable)
+(Include screenshots of the changes if it affects the user interface. This helps reviewers understand the changes better.)
```

Comment:
Burayı tamamen checklist olacak şekilde madde madde ayırmamız daha iyi olabilir. Issue'yu linklememize ihtiyaç kalmıyor çünkü Pull Request üzerinden ilişkilendiriyoruz. Ekran görüntüsü ihtiyacımız da olacağını sanmıyorum. O yüzden tamamen checklistlerden oluşacak şekilde düzenleyebilirsek çok daha kullanışlı olabilir. 

**Örneğin;**
### Checklist

Before submitting your pull request, ensure the following:

- [X] **Title and Branch Naming Conventions:**
    - The pull request title follows the
      standard: [Pull Request Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be/blob/main/CONTRIBUTING.md#pull-request-naming-conventions).
    - The branch name follows one of the conventions outlined in
      the [Branch Naming Conventions](https://github.com/afet-yonetim-sistemi/ays-be/blob/main/CONTRIBUTING.md#branch-naming-conventions).

- [X] **Local Testing:**
    - I have tested my changes locally on Postman, and they are working as expected.

- [X] **Code Quality:**
    - The code is formatted according to the project's coding guidelines and style.
    - The code has been reviewed to ensure its quality.
    - The code does not contain any issues flagged by SonarLint.

- [X] **Documentation:**
    - Necessary documentation has been added or existing documentation has been updated, specifically detailing changes made in Postman.

- [X] **Testing:**
    - Relevant unit tests have been written and included.
    - Relevant integration tests have been written and included.

- [X] **Reviewers and Assignees:**
    - Default reviewers have been assigned to this pull request.
    - Assignees have been added if necessary.

- [X] **Labels and Associations:**
    - No specific actions are required in the Labels and Associations section for this pull request.